### PR TITLE
Init release highlights

### PR DESCRIPTION
**Pull request by anyasabo**
This will need some rework based on:
https://github.com/elastic/cloud-on-k8s/pull/1953

But this is a first draft and can be used to comment on content

### CODE REVIEW

File: docs/release-notes/highlights-1.0.0-beta1.asciidoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+[id="{p}-release-v1beta1"]
+=== v1beta1 version
+
+ECK 1.0.0-beta1 introduces `v1beta1`, a new API version of the various custom resources. The new custom resources are not compatible with previous ECK alpha releases. Existing users must migrate to v1beta1 to use the new ECK features. ECK 1.0.0-beta1 can be installed on top of the current ECK alpha versions. The new version of ECK will leave older resources in place, but will not modify them. To receive all of the benefits of ECK 1.0.0-beta1, we recommend deleting and recreating existing Elasticsearch clusters and other resources. See <<{p}-upgrading-eck>> for more detail.
+
+[float]
+[id="{p}-release-validation"]
+=== Validation changes
+
+In Kubernetes versions <= 1.13, OpenAPI validation of resources on admission is no longer enforced. This is due to a limitation in Custom Resource Definitions, where older versions did not support validation for different versions. We now publish manifests for current versions of Kubernetes, and another for compatibility with older Kubernetes versions.
```
[the remark this answers is not in the document]
Comment (anyasabo) [reply docs/release-notes/highlights-1.0.0-beta1.asciidoc]: Do you think we're sufficiently clear elsewhere that we have two separate CRD flavors now?

**Comment by anyasabo** [comment]
> Perhaps it's worth mentioning the improvements to secure settings, removal of `setVmMaxMapCount`, and addition of default resource limits as well.

For setVmMaxMapCount, it was already mentioned in the upgrading doc (which we link to) so I wasn't sure if it should be called out here as well. Do you think it's worthwhile to highlight?

Agreed on secure settings. I was a little iffy on default limits but that is an ergonomics improvement so I'll add it

**Comment by anyasabo** [comment]
I'm not totally sure how the includes are supposed to work, so could definitely use some advice from an asciidoctor doctor

**Comment by charith-elastic** [comment]
> For setVmMaxMapCount, it was already mentioned in the upgrading doc (which we link to) so I wasn't sure if it should be called out here as well. Do you think it's worthwhile to highlight?

It just feels like something users might trip over. I assume (based on a sample size of one) that more people read release notes than upgrade notes.

**Comment by anyasabo** [comment]
>It just feels like something users might trip over.

Ah I was thinking there's enough other breaking changes there (having to change `nodes` to `nodeSets`, having to specify nodeset names etc that there's no getting around having to read it

File: docs/release-notes/highlights-1.0.0-beta1.asciidoc [hunk 2]
```diff
@@ -1,20 +1,47 @@
...
+
+[float]
+[id="{p}-release-tls"]
+=== More control over TLS settings
+
+Users have more control over TLS settings of the Elasticsearch HTTP layer, Kibana, and APM Server in this release. It is now possible to use external certificates from well-known CAs, utilities like cert-manager, or even completely disable automatic provisioning of TLS certificates. The latter can be useful when TLS is handled at a different layer, for instance, with Istio deployments. See: <<{p}-setting-up-your-own-certificate>> and <<{p}-disable-tls>>. 
+
+[float]
+[id="{p}-release-secure-settings"]
+=== Secure settings enhancements
```
Comment (barkbay) [inline docs/release-notes/highlights-1.0.0-beta1.asciidoc]: Is not also a breaking change? Not sure if it mentioned elsewhere...

Comment (anyasabo) [reply docs/release-notes/highlights-1.0.0-beta1.asciidoc]: You're correct and it is not mentioned in the upgrading-eck doc so I'll open a separate PR for that

File: docs/release-notes/highlights-1.0.0-beta1.asciidoc [hunk 3]
```diff
@@ -1,20 +1,47 @@
...
+=== Default limits
 
-TODO
+Default limits are now set for the various resource types if none are explicitly specified. Previously, if left at the default, Pods would consume unbounded amounts of memory, which could lead them to be OOM killed. This allows for a better experience in demo scenarios. We recommend in production that both requests and limits be set appropriately. See <<{p}-managing-compute-resources>> for more information.
+
+[float]
+[id="{p}-breaking-changes-1.0.0-beta1"]
+=== Breaking Changes
 
 Attempting to delete resources created with a 0.9.0 version will hang if ECK 1.0.0-beta1 is running. To unblock the deletion, remove any registered finalizer from the resource:
```
Comment (pebrc) [inline docs/release-notes/highlights-1.0.0-beta1.asciidoc]: feels somehwhat out of place to have kubectl commands in the release highlights doc

File: docs/release-notes/highlights-1.0.0-beta1.asciidoc [hunk 4]
```diff
@@ -1,20 +1,47 @@
...
+[id="{p}-release-tls"]
+=== More control over TLS settings
+
+Users have more control over TLS settings of the Elasticsearch HTTP layer, Kibana, and APM Server in this release. It is now possible to use external certificates from well-known CAs, utilities like cert-manager, or even completely disable automatic provisioning of TLS certificates. The latter can be useful when TLS is handled at a different layer, for instance, with Istio deployments. See: <<{p}-setting-up-your-own-certificate>> and <<{p}-disable-tls>>. 
+
+[float]
+[id="{p}-release-secure-settings"]
+=== Secure settings enhancements
+
+Secure settings have received multiple ergonomics improvements. It is now possible to specify secure settings from multiple secrets. It is also possible to specify specific items in a Kubernetes secret to be used as a secure setting, rather than having to use all items in a secret.
```
Comment (pebrc) [inline docs/release-notes/highlights-1.0.0-beta1.asciidoc]: 'ergonomics' made me laugh

Comment (anyasabo) [reply docs/release-notes/highlights-1.0.0-beta1.asciidoc]: I guess it is an "ergonomic" improvement? Language is weird

File: docs/release-notes/highlights-1.0.0-beta1.asciidoc (hunk 3 above)
Comment (anyasabo) [reply docs/release-notes/highlights-1.0.0-beta1.asciidoc]: That's fair, I'll move the actual steps to the upgrading doc and link to it

**Pull request merged**